src/modules/ui/components/layout/dimensions.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+
+import { baseFonts } from '../theme';
+
+const container = {
+  position: 'absolute',
+  padding: 5,
+  bottom: 10,
+  right: 10,
+  backgroundColor: 'rgba(255, 255, 255, 0.5)',
+};
+
+const dimensionStyle = {
+  fontSize: 12,
+  ...baseFonts,
+};
+
+const delimeterStyle = {
+  margin: '0px 5px',
+  fontSize: 12,
+  ...baseFonts,
+};
+
+// Same as Chrome's timeout in the developer tools
+const DISPLAY_TIMEOUT = 1000;
+
+class Dimensions extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      isVisible: false,
+    };
+  }
+
+  componentWillReceiveProps({ width, height }) {
+    if (width !== this.state.width || height !== this.state.height) {
+      this.onChange(width, height);
+    }
+  }
+
+  onChange(width, height) {
+    this.setState({ isVisible: true });
+
+    setTimeout(() => {
+      // Ensure the dimensions aren't still changing
+      if (width === this.props.width && height === this.props.height) {
+        this.setState({ isVisible: false });
```

Comment:
This will potentially be called after the component unmounts, which will throw a react warning/error.

You should keep a reference to the timeout on this class and clear it before unmount.

```js
class Dimensions extends React.Component {
  _hideTimeout = null;
  ...
  componentWillUnmount() {
    clearTimeout(this._hideTimeout);
  }
  ...
  onChange(...) {
    this._hideTimeout = setTimeout(() => { ... }, DISPLAY_TIMEOUT);
  }
}
```